### PR TITLE
revert message

### PR DESCRIPTION
**Pull request by supernovahs**
Solves  https://github.com/paradigmxyz/reth/issues/3631

### CODE REVIEW

**Comment by supernovahs** [comment]
Hey @mattsse , Was my code buggy ?😇
Anyway all good, just new to rust.

File: crates/revm/revm-inspectors/src/tracing/types.rs [hunk 1]
```diff
@@ -161,7 +161,14 @@ impl CallTrace {
...
     pub(crate) fn as_error(&self) -> Option<String> {
-        self.is_error().then(|| format!("{:?}", self.status))
+        if self.is_error() {
+            match self.status {
+                InstructionResult::Revert => Some("Reverted".to_string()),
+                _ => Some(format!("{:?}", self.status)),
+            }
+        } else {
+            None
+        }
```
Comment (mattsse) [inline crates/revm/revm-inspectors/src/tracing/types.rs:171]: I liked this better
```suggestion
 self.is_error().then(|| match self.status {
            InstructionResult::Revert => format!("Reverted"),
            status => format!("{:?}", status),
        })
```

**Review by mattsse — CHANGES_REQUESTED** [review]
no it wasn't was just formatting manually

**Comment by mattsse** [comment]
ah my bad, there are two PRs for the same thing #3640 

got me confused

**Comment by supernovahs** [comment]
Alright . I am closing this PR then . Thanks

**Pull request closed without merging**